proxy/proxy.go

```diff
@@ -117,7 +117,8 @@ func NewProxy(c Config) (*Proxy, error) {
 func (proxy *Proxy) AttachExistingContainers() {
 	containers, _ := proxy.client.ListContainers(docker.ListContainersOptions{})
 	for _, cont := range containers {
-		if strings.HasPrefix(cont.Command, weaveWaitEntrypoint[0]) {
+		if strings.HasPrefix(cont.Command, weaveWaitEntrypoint[0]) ||
+			strings.HasPrefix(cont.Command, "/home/weave/weaver") {
```

Comment:
wouldn't it make more sense to check the image name, since that is how we make the same decision in `ContainerStarted`?